pkgs/top-level/all-packages.nix

```diff
@@ -608,7 +608,9 @@ in
 
   makeBinaryWrapper = makeSetupHook {
     deps = [ dieHook ];
-  } ../build-support/setup-hooks/make-binary-wrapper.sh;
+  } (runCommand "make-binary-wrapper.sh" {} ''
+      substitute ${../build-support/setup-hooks/make-binary-wrapper.sh} $out --replace " cc " " ${gcc}/bin/cc "
```

Comment:
I think it'd be safer to use `@cc@ instead of this substitution.

Reply:
done